testament/testament.nim

```diff
@@ -401,8 +401,7 @@ proc codegenCheck(test: TTest, target: TTarget, spec: TSpec, expectedMsg: var st
   try:
     let genFile = generatedFile(test, target)
     let contents = readFile(genFile).string
-    let check = spec.ccodeCheck
-    if check.len > 0:
+    for check in spec.ccodeCheck:
```

Comment:
Why remove empty check?

Reply:
because semantics changed from a string to a seq[string]; if ccodeCheck.len == 0, that block will be skipped

Reply:
What about `@["", "", ""]`

Reply:
you're right, I changed to `if check.len > 0 and check[0] == '\\':`